sdk/tools/cabman/dfp.cxx

```diff
@@ -498,7 +489,7 @@ bool CDFParser::SetDiskName(PCABINET_NAME *List, ULONG Number, char* String)
         CN = CN->Next;
     }
 
-    CN = (PCABINET_NAME)malloc(sizeof(CABINET_NAME));
+    CN = new CABINET_NAME;
```

Comment:
You could get slightly fancier, and have a constructor with arguments that take the DiskNumber and the "Name" parameters, that are otherwise initialized just below.

Reply:
not looking to create the diff as big as possible, just to make the std::string work.
calling malloc on an object with an std::string in it is not going to work well ;)